packages/core-react/src/components/Card/index.js

```diff
@@ -2,10 +2,34 @@ import React from 'react';
 import clsx from 'clsx';
 import PropTypes from 'prop-types';
 
-export default function Card({ heading, body, img, row }) {
+import rayExports from '@wework/ray-core/scss/ray-exports.scss';
```

Comment:
Can we instead add this to the javascript settings and import from there?


https://github.com/wework/ray/blob/master/packages/core/src/global/js/settings.js

Reply:
ahh, didnt know about that. will look into it. Although, dont see any advantage in sharing info between css/jss this way? to reduce duplication.